### ISSUE TITLE
ReactJS

### ISSUE THREAD

**Issue by aldoademi** [thebaselab/codeapp#801]
Is ReactJS supported?
When I try this command:
`npm start`
Return this error:
`npm ERR! Command doesn’t exist or is not supported: react-scripts start`

**Comment by bummoblizard** [comment]
Yes. But certain Cli commands might not be supported. Please refer to the React template for a sample setup: https://github.com/thebaselab/react-starter

**Comment by aldoademi** [comment]
I Started this project from a pc. Now what can I do, i didn't understand well

**Comment by aldoademi** [comment]
Now I have this error:
`Programma interrupted. This could be caused by a memory limit or an error in your code`

**Comment by DroneBreaker** [comment]
Has the issue been solved?
